docs/astro/src/content/docs/reference/primitive-types.mdx

```diff
@@ -71,6 +71,17 @@ export component CharacterCountOfString {
 }
 ```
 
+The `to-lower` and `to-upper` methods convert `string` to lowercase or uppercase according to the [Unicode Character Property](https://www.unicode.org/versions/Unicode16.0.0/core-spec/chapter-4/#G124722).
```

Comment:
Great work - thanks!

I'd like to suggest to lengthen the names. "upper" and "lower" on a string by itself makes no sense IMO. As it happens, capital letters (the metal objects) were stored in the (literal) upper case, where this originates from. I think the function name as such should be complete, i.e. `to-upper-case()` and `to-lower-case()`.

Reply:
It should be `to-uppercase` and `to-lowercase`  to be consistent with our C++/Rust API.

Reply:
That works for me :)